### ISSUE TITLE
bloom populating patches branch in release repo with .travis.yml

### ISSUE THREAD

**Issue by asmodehn** [ros-infrastructure/bloom#400]
Currently, when hooking up a release repository build by bloom to travis, all branches are built, but this is useless.

A simple travis build configuration can be added to all the branches that are meaningless for the user ( like `patches/.*` ) to prevent travis from building them, in the event that a user want to test his packages with travis : 
```
sudo: required
dist: trusty
language: generic

branches:
  except:
    - /^patches.*$/ # no point in building patches branches

notifications:
  email: false
```

Note it is especially useful to test a release repo with travis when doing third party releases.

The upstream source repo can be tested on his own with some specific travis rules, when using from source (multiple python environment for example).

But the release repo can be patched to do ROS testing of the packages( catkin build + catkin tests ) for each distribution it is released in. note I am not talking about testing that making the deb package works (job of ros buildfarm) but rather verifying that the tests of the package itself pass in a ROS environment.

**Comment by asmodehn** [comment]
I just found the travis setting to turn off building if there is no .travis.yml present.
Much better.

I guess the only improvement I can wish for here is the other side of the problem : having bloom generating "generic ROS testing configuration on travis". Anyway it s another problem.

For future reference, Here are some of the repositories where I am playing with thirdparty travis testing : 
- https://github.com/asmodehn/pyzmp-rosrelease
- https://github.com/asmodehn/pyros-config-rosrelease

Closing this issue as there already is an obvious enough solution.
